knative/client.go

```diff
@@ -16,8 +17,45 @@ const (
 	DefaultWaitingTimeout = 60 * time.Second
 )
 
-func NewClient(namespace string, verbose bool) (clientservingv1.KnServingClient, io.Writer, error) {
+func NewServingClient(namespace string, verbose bool) (clientservingv1.KnServingClient, io.Writer, error) {
 
+	knParams := initKnParams(verbose)
+
+	client, err := knParams.NewServingClient(namespace)
```

Comment:
You should use the constructor from the client API package to avoid a link into the command line parsing.

Not sure how easy this is, but we should make this possible.